test-graphql/product-helpers-test.js

```diff
@@ -0,0 +1,38 @@
+import assert from 'assert';
+import Config from '../src-graphql/config';
+import Client from '../src-graphql/client';
+import ProductHelpers from '../src-graphql/product-helpers';
+import singleProductFixture from '../fixtures/product-fixture';
+import fetchMock from './isomorphic-fetch-mock'; // eslint-disable-line import/no-unresolved
+
+suite('product-helpers-test', () => {
+  const productHelpers = new ProductHelpers();
+  const config = new Config({
+    domain: 'single-product.myshopify.com',
+    storefrontAccessToken: 'abc123'
+  });
+
+  const client = new Client(config);
+
+  test('it returns the variant based on options given', () => {
+    fetchMock.postOnce('https://single-product.myshopify.com/api/graphql', singleProductFixture);
```

Comment:
As a separate improvement, we should be able to use a fixture plus it's query and decode it to get the `product` graph model result *without* having to go through a client and do fetch mocking. Ideally in a unit test we can avoid all of that.

Something like:

```js
const product = decode(query, singleProductFixture); // or even a fixture to import which has already done this
const variant = productHelpers.variantForOptions(product, {Fur: 'Fluffy', Size: 'Medium'});

assert.equal(variant.id, 'gid://shopify/ProductVariant/25602235976');
```